infracost-usage-example.yml

```
@@ -170,6 +171,10 @@ resource_usage:
   google_dns_record_set.my_record_set:
     monthly_queries:  1000000 # Monthly DNS queries.
 
+  google_kms_crypto_key.my_keys:
+    monthly_key_versions: 10000 # Number of uses crypto key versions per month
```

```suggestion
    key_versions: 10000 # Number of key versions.
```

From the contributing guide's "Usage file notes" section, number 5 would apply here since each key version is used continuously in the month. You'll need to tweak the resource go file to use the new name too.
5. For resources that are continuous in time, do not use prefixes, e.g. use `instances`, `subscriptions`, `storage_gb`. For non-continuous resources, prefix with `monthly_` so users knows what time interval to estimate for, e.g. `monthly_log_lines`, `monthly_requests`.